internal/cluster/partition.go

```diff
@@ -119,9 +119,10 @@ func StartZenPartitionNode(mainCtx context.Context, cfg *config.RqLite) (*ZenPar
 				b.WriteString(fmt.Sprintf("failed to download auto-restore file: %s", err.Error()))
 				if errOK {
 					b.WriteString(", continuing with node startup anyway")
-					log.Info(b.String())
+					log.Info(b.String(), nil)
```

Comment:
nil arg is not needed when there are no formatting directives.

Reply:
Well, the Go compiler has another opinion - when I remove the parameter this failure is reported and the tests do fail (and likely the app will not run). 
This is true for all the for changes, where I added "nil".
I'm open for alternative solutions.
```
go test ./...
internal/cluster/partition.go:122:15: non-constant format string in call to github.com/pbinitiative/zenbpm/internal/log.Info
FAIL    github.com/pbinitiative/zenbpm/internal/cluster [build failed]
```

Using `go version go1.24.0 darwin/arm64`